modules/creating-and-exporting-an-api-token.adoc

```diff
@@ -0,0 +1,8 @@
+// Module included in the following assemblies:
+//
+// * cli/getting-started-cli.adoc
+:_content-type: CONCEPT
+[id="creating-and-exporting-an-api-token_{context}"]
+= Creating and exporting an API Token
+
+Create an API token and store it securely. You can either export the token or save it to a file with the `roxctl central debug dump` command.
```

Comment:
Hmm, I don't think we should recommend people to use `roxctl central debug dump` for those purposes.

Instead, obtaining the token should always either be done via UI or API. I think in general we don't really provide API examples within our docs (and there's currently no `roxctl central api-token create` command or anything like that available), so let's just leave this one out for now?

Background is that the debug dump should really only be used in combination with support investigations, e.g. support requesting it on an opened case. Don't think we want this in place for such potential frequent tasks.